_integration-test/run.sh

```diff
@@ -28,7 +28,12 @@ teardown() {
   DID_TEAR_DOWN=1
 
   if [ "$1" != "EXIT" ]; then
-    echo "An error occurred, caught SIG$1 on line $2"
+    set +x
+    error_msg="An error occurred, caught SIG$1 on line $2"
+    echo "$error_msg"
+    dsn="https://5a620019b5124cbba230a9e62db9b825@o1.ingest.us.sentry.io/6627632"
+    local sentry_cli="docker run --rm -v $(pwd):/work -e SENTRY_DSN=$dsn getsentry/sentry-cli"
```

Comment:
Odd to me that we set REPORT_ISSUES=0 above, but then send an envelope here anyway? Maybe I am misunderstanding something, but why would we not just do `REPORT_ISSUES=1`, and maybe figure out some way to configure it to send to an "integration tests" or similar project on our actual dogfood instance?

Reply:
Sure, I can incorporate this into the existing logic for `REPORT_ISSUES` I believe.